src/aura/dataTable/dataTable.css

```diff
@@ -2,4 +2,8 @@
   cursor: pointer;
   color: rgb(0, 95, 178);
   display: block;
-}
\ No newline at end of file
+}
+
+.THIS.accounts-data {
```

Comment:
if there is no styles for this rule, get rid of it

Reply:
Done